core/app/models/concerns/spree/user_address_book.rb

```diff
@@ -102,7 +106,6 @@ def remove_from_address_book(address_id)
         user_address = user_addresses.find_by(address_id: address_id)
         if user_address
           user_address.update_attributes(archived: true, default: false)
```

Comment:
I know this isn't from this PR but should this be a bang version of `update_attributes!`?